common/static/common/js/discussion/utils.js

```diff
@@ -539,6 +542,37 @@
             };
         };
 
+        DiscussionUtil.handleKeypressInToolbar = function(event) {
+                var $currentButton, $nextButton, $toolbar, $allButtons,
+                    keyPressed, nextIndex, currentButtonIndex;
+
+                var leftKey = this.LEFT_KEYCODE;
```

Comment:
Why do we want to assign those constants to local variables? Why not just use them in the statements below?

Reply:
I just thought it made the code look a little more readable this way